scipy/stats/_multivariate.py

```diff
@@ -3394,10 +3394,7 @@ def rvs(self, dim, size=1, random_state=None):
             x[0] += D[n]*np.sqrt(norm2)
             x /= np.sqrt((norm2 - x0**2 + x[0]**2) / 2.)
             # Householder transformation
-            Hx = (np.eye(dim-n) - np.outer(x, x))
-            mat = np.eye(dim)
-            mat[n:, n:] = Hx
-            H = np.dot(H, mat)
+            H[:, n:] -= np.outer(np.dot(H[:, n:], x), x)
```

Comment:
Does it need `np.outer`? It's scalar-vector product, is it not?

Reply:
Unless I'm missing another simplification, I think this does need outer. The result of `np.dot(H[:, n:], x)` is a vector and you want the outer product of that vector and `x`.